### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by pmaxy**
COMMITS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+- Add a feature branch
+- Add linter file
+- Add a gitignore file
+- Add Webhint
+- Add stylelintrc
+- Add new Html file
+- Add test.md file
+- ignore test file
+# AUTHOR
+-@PATRICK WANJIRU
```
Comment (Yerimah) [inline README.md]: - [ ] Make sure you add a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project. A README file provides an introduction to the files contained in the repository and they prevent a person viewing or using your repository from needing to read your mind.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="G:\GIT WORKSPACE\HELLO_WORLD_PROJECT\style.css" >
+        <title><h1 class="color1">Hello Microverse!</h1></title>
+    </head>
+        <body>
+
+
+        </body>
+</html>
```
Comment (Yerimah) [inline index.html]: - [ Required] You do not have any code in the body of your HTML file. Please add a `<h1>` tag that displays `Hello Microverse`
- Your external stylesheet is not properly linked in the head of your HTML file. Reference only `style.css` as it is the name of your stylesheet

File: .hintrc [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (Yerimah) [inline .hintrc:1]: - [ ] Your linters are failing. Linter result should be green, this means that the last commit have a green check mark (✅) or a summary of the passed checks is in the PR.
- [ ] Please add a descriptive title and a short summary to your pull request. In order to make your repo look professional, pull request should have a descriptive title telling you what changes are inside and pull request should have a short summary describing introduced changes in more detail.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#color2 {
+    color: red;
+  }
+
```
Comment (Yerimah) [inline style.css]: - [ ] Hey @pmaxy 👋 you should also make sure that the CSS id selector is existing in your HTML file.

**Review by Yerimah — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️ 

Hi @pmaxy , 

Great work on making the changes requested by a previous reviewer 👏 but there are still some which aren't addressed yet.

So Close Snl GIF by Saturday Night Live

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 4 above)
Comment (pmaxy) [reply style.css]: Thank you @Yerimah I have tried to update every section of my files

**Comment by pmaxy** [comment]
@Yerimah Thank you for the review

File: index.html [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="G:\GIT WORKSPACE\HELLO_WORLD_PROJECT\style.css" >
+        <title>Welcome Microverse </title>
+    </head>
+        <body>
+            <h1 class="color1">Hello Microverse!</h1>
```
Comment (Nyame-Wolf) [inline index.html:10]: [Optional] This is a class. In the css you used `#color1` which is an id selector instead of `.color1` the class selector.

**Review by Nyame-Wolf — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @pmaxy  :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :sparkles:

## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by pmaxy** [comment]
@Yerimah  thank you alot for approving on css i will change

**Comment by pmaxy** [comment]
Merge the feature branch to development

**Pull request merged**